etl/RetrosheetParser.py

```diff
@@ -24,14 +24,10 @@ def __init__(self):
                 , "padj": self.parse_padj
                 }
         
-        info_types = ['visteam', 'hometeam', 'site', 'date', 'number', 'starttime', 'daynight', 'usedh', 'umphome', 'ump1b', 'ump2b', 'ump3b', 'howscored', 'pitches', 'temp', 'winddir', 'windspeed', 'fieldcond', 'precip', 'sky', 'timeofgame', 'attendance', 'wp', 'lp', 'save', 'gwrbi']
-        
-        play_types = tuple(['S', 'K', 'D', 'H', 'W', 'IW', 'T', 'E', 'FC', 'HP', 'C', 'NP'] + map(str,range(1,10)))
```

Comment:
This deletion really comes as a result of my opinion that we shouldn't eliminate data at this stage. I think the first pass should be as light of a transformation as possible to get it into a relational database. After that we can filter out data that isn't strictly relevant if it becomes cumbersome or confusing for analysis.